pre-docs/inventory_version2.md

```diff
@@ -541,6 +541,80 @@ plugins:
 * `private-key`: The path to the private key file. default: `keys/private_key.pkcs7.pem`
 * `public-key`: The path to the public key file. default: `keys/public_key.pkcs7.pem`
 
+#### Vault plugin
+
+
+This plugin allows config values to be set by accessing secrets on a Vault server. It supports several fields:
+
+- `_plugin`: The value of `_plugin` must be `vault`
+- `server_url`: The URL of the Vault server (optional, defaults to `ENV['VAULT_ADDR']`)
+- `auth`: The method for authorizing with the Vault server and any necessary parameters (optional, defaults to `ENV['VAULT_TOKEN']`)
+- `path`: The path to the secrets engine (required)
+- `field`: The specific secret being used (optional, defaults to a Ruby hash of all secrets at the `path`)
+
+
+**Authentication Methods**
+
+Vault requires a token to assign an identity and set of policies to a user before accessing secrets. Currently, the Vault plugin offers 2 authentication methods:
+
+
+`token`
+
+Authenticate using a valid token. This method requires the following fields:
+
+- `method`: The value of `method` must be `token`
+- `token`: The token to authenticate with
+
+
+`userpass`
+
+Request a token by logging into the Vault server with a username and password. This method requires the following fields:
+
+- `method`: The value of `method` must be `userpass`
+- `user`: The username
+- `pass`: The password
+
+
+Each of the available fields for the Vault plugin can be used in the inventory configuration:
```

Comment:
Would it be equally clear if we shortened and changed to active voice, like this:

`You can add any Vault plugin field to the inventory configuration:`